two issues:

1. Tests don't get executed
2. Build fails with:

```
:publishNebulaPublicationToMavenLocal FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':publishNebulaPublicationToMavenLocal'.
> Failed to publish publication 'nebula' to repository 'mavenLocal'
   > Invalid publication 'nebula': artifact file does not exist: '/home/travis/build/maslick/barkoder/build/libs/barkoder-0.0.1-SNAPSHOT.jar'
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 19s
9 actionable tasks: 9 executed
The command "./gradlew clean bintrayUpload" exited with 1.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,4 +13,4 @@ cache:
     - $HOME/.gradle/wrapper/
 
 script:
-  - ./gradlew clean bintrayUpload
+  - ./gradlew clean build bintrayUpload

--- a/build.gradle
+++ b/build.gradle
@@ -93,7 +93,4 @@ bintray {
         licenses = ['MIT']
         vcsUrl = 'https://github.com/maslick/barkoder.git'
     }
-    version {
-        name = "$version"
-    }
 }

--- a/build.gradle
+++ b/build.gradle
@@ -32,7 +32,7 @@ apply plugin: 'nebula.source-jar'
 apply plugin: 'com.jfrog.bintray'
 
 group = 'io.maslick'
-version = '0.0.1-SNAPSHOT'
+version = '0.1-SNAPSHOT'
 sourceCompatibility = 1.8
 
 compileKotlin {
@@ -90,5 +90,10 @@ bintray {
     pkg {
         repo = 'maven'
         name = 'barkoder'
+        licenses = ['MIT']
+        vcsUrl = 'https://github.com/maslick/barkoder.git'
+    }
+    version {
+        name = "$version"
     }
 }

--- a/build.gradle
+++ b/build.gradle
@@ -89,6 +89,6 @@ bintray {
     override = true
     pkg {
         repo = 'maven'
-        name = 'collector'
+        name = 'barkoder'
     }
 }

